Documented Django Admin Log Entries in v2.1 Release Overview. (#4953)
nautobot/nautobot: nautobot/docs/release-notes/version-2.1.md

!!! tip
    Users must have permission to `view` the `extras > file proxy` object type in order to list and download files from the REST API.

#### Django Admin Log Entries ([#4646](https://github.com/nautobot/nautobot/pull/4646))

> Django Admin Log Entries are implemented to record administrative changes made under the "Admin" section of the user interface. Changes (add/delete/update) to Objects like Users, Group, Object Permissions, and etc. in the "Admin" user interface will now be recorded and displayed as "Log Entries" under the "Administration" section of the Admin List View.

Reviewer: Suggesting some wording changes. Might be worth calling out that the log entries were already present, this is just adding a UI to view them?

```suggestion
Django Admin Log Entries record administrative changes made under the "Admin" section of the user interface. Changes (add/delete/update) to Objects like Users, Group, Object Permissions, etc. in the "Admin" user interface are now displayed as "Log Entries" under the "Administration" section of the Admin UI.
```
